buildenv/jenkins/common/build.groovy

```diff
@@ -288,7 +288,11 @@ def archive_sdk() {
         def testDir = "test"
 
         dir(OPENJDK_CLONE_DIR) {
-            sh "tar -C ${buildDir} -zcvf ${SDK_FILENAME} ${JDK_FOLDER}"
+            // The archiver receives pathnames on stdin and writes to stdout.
+            def archiveCmd = SPEC.contains('zos') ? 'pax -wvz' : 'tar -cvz -T -'
+            // Filter out unwanted files (most of which are available in the debug-image).
+            def filterCmd = "sed -e '/\\.dbg\$/d' -e '/\\.debuginfo\$/d' -e '/\\.diz\$/d' -e '/\\.dSYM\\//d' -e '/\\.map\$/d' -e '/\\.pdb\$/d'"
```

Comment:
There's a `//` before the `d` in the `.dSYM` case:
`-e '/\\.dSYM\\//d'` -> `-e '/\\.dSYM\\/d'`

Reply:
That's intentional (to match `.dSYM/`).

Reply:
And looking at the osx build, it's clearly working